UT-IfcCourse/UT-IfcCourse.ifc

```diff
@@ -0,0 +1,56 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION (('ViewDefinition[Ifc4x3NotAssigned]'), '2.1');
+FILE_NAME ('UT-IfcCourse.ifc', '2021-02-17T09:25:56', (), ('AKG Software Consulting GmbH'), $, $, $);
+FILE_SCHEMA (('IFC4X3_RC2'));
+ENDSEC;
+DATA;
+#1 = IFCORGANIZATION('AKG Software Consulting GmbH', 'AKG Software Consulting GmbH', $, $, $);
+#2 = IFCAPPLICATION(#1, 'Build58', 'VESTRA INFRAVISION', 'VESTRA INFRAVISION');
+#3 = IFCPERSON($, 'VESTRA', 'VESTRA', $, $, $, $, $);
+#4 = IFCPERSONANDORGANIZATION(#3, #1, $);
+#5 = IFCOWNERHISTORY(#4, #2, $, $, $, $, $, 1613550356);
+#6 = IFCDIMENSIONALEXPONENTS(0, 0, 0, 0, 0, 0, 0);
+#7 = IFCUNITASSIGNMENT((#8, #9, #10));
+#8 = IFCSIUNIT(*, .LENGTHUNIT., $, .METRE.);
+#9 = IFCSIUNIT(*, .TIMEUNIT., $, .SECOND.);
+#10 = IFCSIUNIT(*, .PLANEANGLEUNIT., $, .RADIAN.);
+#11 = IFCAXIS2PLACEMENT3D(#12, $, $);
+#12 = IFCCARTESIANPOINT((0., 0., 0.));
+#13 = IFCGEOMETRICREPRESENTATIONCONTEXT($, 'Model', 3, 1.E-6, #11, $);
+#14 = IFCPROJECTEDCRS('EPSG:25832', 'ETRS89_UTM32', 'EPSG:25832', 'EPSG:25832', 'UTM', '32', #8);
+#15 = IFCMAPCONVERSION(#13, #14, 0., 0., 0., $, $, $);
+#16 = IFCLOCALPLACEMENT($, #17);
+#17 = IFCAXIS2PLACEMENT3D(#18, $, $);
+#18 = IFCCARTESIANPOINT((51.888, -7.639, 0.));
+#19 = IFCPROJECT('1W7lFkDRwMWKYdP6AbIlsD', #5, 'Projekt: UT-IfcCourse', 'Projekt: UT-IfcCourse', $, $, $, (#13), #7);
+#20 = IFCSITE('1d1nf$UMdHWK5jSGoE5PmO', #5, 'VESTRA', 'VESTRA', $, #16, $, $, $, $, $, $, IFCLABEL('VESTRA'), $);
+#21 = IFCRELAGGREGATES('21OxTI03ezZaR79ypFQCSS', #5, $, $, #19, (#20));
+#22 = IFCRELCONTAINEDINSPATIALSTRUCTURE('2ue$NsAuR7lKPQjvj1KAR1', #5, $, $, (#31, #35), #20);
+#23 = IFCPROPERTYSET('0zF$PjViSGYq3EgDM4iXUP', $, 'ePSet_MapConversion', $, (#24, #26, #28));
+#24 = IFCPROPERTYSINGLEVALUE('Eastings', $, IFCREAL(0.), $);
+#25 = IFCSIUNIT(*, .LENGTHUNIT., $, .METRE.);
+#26 = IFCPROPERTYSINGLEVALUE('Northings', $, IFCREAL(0.), $);
+#27 = IFCSIUNIT(*, .LENGTHUNIT., $, .METRE.);
+#28 = IFCPROPERTYSINGLEVALUE('OrthogonalHeight', $, IFCREAL(0.), $);
+#29 = IFCSIUNIT(*, .LENGTHUNIT., $, .METRE.);
+#30 = IFCRELDEFINESBYPROPERTIES('311fBqOpiYk4mhf4rEK3nO', #5, $, $, (#19), #23);
+#31 = IFCALIGNMENT('1iESu5BcMzf4oUgFXveuh1', $, 'Achse Nr. 1', 'Achse Nr. 1', $, #16, $, #32, $);
+#32 = IFCPOLYLINE((#33, #34));
+#33 = IFCCARTESIANPOINT((513210., 5454110., 91.));
+#34 = IFCCARTESIANPOINT((513220., 5454120., 92.));
+#35 = IFCCOURSE('1Y6olD18bsgajKcDf8M1yx', $, 'Querprofile Achse 1', 'Querprofile Achse 1', $, #16, #36, $, $);
+#36 = IFCPRODUCTDEFINITIONSHAPE($, $, (#37));
+#37 = IFCSHAPEREPRESENTATION(#38, 'Body', 'Curve2D', (#39));
+#38 = IFCREPRESENTATIONCONTEXT(IFCLABEL('Curve2D'), IFCLABEL('Curve2D'));

```

Comment:
IfcRepresentationContext is an abstract class, and only subtypes can be instantiated.